RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletEDProducerT.cc

```diff
@@ -86,21 +86,19 @@ void CAHitNtupletEDProducerT<T_Generator>::produce(edm::Event& iEvent, const edm
   generator_.initEvent(iEvent, iSetup);
 
   LogDebug("CAHitNtupletEDProducer") << "Creating ntuplets for " << regionDoublets.regionSize() << " regions, and " << regionDoublets.layerPairsSize() << " layer pairs";
-
-  typename T_Generator::ResultType ntuplets;
-  ntuplets.reserve(localRA_.upper());
-
+  std::vector<typename T_Generator::ResultType> ntuplets;
+  ntuplets.resize(regionDoublets.regionSize());
+  for(auto ntuplet : ntuplets)  ntuplet.reserve(localRA_.upper()); 
```

Comment:
maybe use ``` auto& ``` or ```auto&& ```
 to be a bit more obvious that it's not a temp copy that's modified.